content/tracing/languages/java.md

```diff
@@ -0,0 +1,274 @@
+        try {
+          scope.span().setTag(DDTags.SERVICE_NAME, "my-new-service");
+
+          // The code you're tracing
+          Thread.sleep(1000);
+
+          // If you don't call close(), the span data will NOT make it to Datadog!
+        finally {
+          scope.close();
+        }
+    }
+}
+```
+
+Alternatively, wrap the code you want to trace in a `try-with-resources` statement:
+
+```java
+import io.opentracing.util.GlobalTracer;
+
+class InstrumentedClass {
+
+    void method0() {
+        Tracer tracer = GlobalTracer.get();
+
+        try (Scope scope = tracer.buildSpan("operation-name").startActive(true)) {
+            scope.span().setTag(DDTags.SERVICE_NAME, "my-new-service");
+            Thread.sleep(1000);
+        }
+    }
+}
+```
+
+In this case, you don't need to call `scope.close()`.
+
+If you’re not using `dd-trace-java.jar`, you must provide a configured tracer. This can be easily done by using the `TracerFactory` or manually in the bootstrap method (i.e. `main`).
+
+```java
+import datadog.opentracing.DDTracer;
+import io.opentracing.util.GlobalTracer;
+import datadog.trace.api.sampling.AllSampler;
+
+public class Application {
+
+    public static void main(String[] args) {
+
+        // Initialize the tracer from environment variables or system properties
+        Tracer tracer = new DDTracer();
+        GlobalTracer.register(tracer);
+
+        // OR from the API
+        Writer writer = new DDAgentWriter();
+        Sampler sampler = new AllSampler();
+        Tracer tracer = new DDTracer(writer, sampler);
+        GlobalTracer.register(tracer);
+
+        // ...
+    }
+}
+```
+
+Notice the above examples only use the OpenTracing classes. Please reference the [OpenTracing API](https://github.com/opentracing/opentracing-java) for more details and information.
+
+## Sampling / distributed tracing
+
+Priority sampling ensures that distributed traces are complete by assigning and propagating a priority value along each trace.
+
+Current Priority Values (more may be added in the future):
+
+|Sampling Value | Effect                                                                                                     |
+|---------------|:----------------------------------------------------------------------------------------------------------:|
+|SAMPLER_DROP   | The sampler automatically decided to not keep the trace. The Agent will drop it.                           |
+|SAMPLER_KEEP   | The sampler automatically decided to keep the trace. The Agent will keep it. Might be sampled server-side. |
+|USER_DROP      | The user asked to not keep the trace. The Agent will drop it.                                              |
+|USER_KEEP      | The user asked to keep the trace. The Agent will keep it. The server will keep it too.                     |
+
+
+Priority sampling is disabled by default. To enable it, configure the `priority.sampling` flag to `true`. [How to configure the client?](# Configuration).
```

Comment:
should be `#configuration` without the space no?

Reply:
👍